src/OmniSharp.Abstractions/Endpoints.cs

```diff
@@ -64,5 +59,9 @@ public static class V2
             public const string GetCodeActions = "/v2/getcodeactions";
             public const string RunCodeAction = "/v2/runcodeaction";
         }
+        
+        public const string GetTestStartInfo = "/getteststartinfo";
+        
+        public const string RunDotNetTest = "/rundotnettest";
```

Comment:
I'm thinking this should go under V2 as well, because it is new.

Also the name, seems very specific, couldn't we name it simply `/v2/runtest`?